public/app/views/pdf/_titlepage.html.erb

```diff
@@ -1,4 +1,5 @@
 <div class="title-page">
+    <div class="logo"><%= image_tag( record.resolved_repository['image_url'] || asset_url("Aspace-logo.png") ) %></div>
```

Comment:
@sdm7g Can you change the filename from Aspace-logo.png to archivesspace.small.png? Aspace-logo.png is an old image that needs to be removed from the codebase.

Reply:
I pushed that change to anw-244 branch, and it's showing up on that branch. In the past I would also see that commit reflected here in the pull request, but it's not.  Is there something else I have to do to refresh ? ( In the past pull req. were coming from my own fork -- does that work differently? )